src/components/FilterButton.js

```diff
@@ -1,29 +1,21 @@
 import React from 'react';
 import styled, { css } from 'styled-components';
 
-const FilterButtons = ({
-  filterPlant,
-  plantCardsFiltered,
-  savedPlants,
-  setFilteredPlants,
-}) => {
+const FilterButtons = ({ filterPlant, plantCardsFiltered }) => {
   return (
     <>
-      <FilterButton
-        variant="all"
-        onClick={() => setFilteredPlants(savedPlants)}
-      >
+      <FilterButton variant="all" onClick={() => filterPlant('')}>
         All Plants
       </FilterButton>
-      <div>
+      <Container>
         {plantCardsFiltered?.map((category, index) => {
           return (
             <FilterButton onClick={() => filterPlant(category)} key={index}>
```

Comment:
Maybe `category` would be a better `key` here since it is always unique :) 
```suggestion
            <FilterButton onClick={() => filterPlant(category)} key={category}>
```